script.js

```diff
@@ -1,4 +1,226 @@
+      guess != "reversed paper" ||
+      guess != "reversed rock"
+    ) {
+      myOutputValue =
+        "Please try again and input the following : Scissors, Paper, Rock, Reversed Scissors, Reversed Paper, Reversed Rock";
+    }
+    var randomChoice = generateRandomChoice();
+    var randomChoiceReversed = generateRandomChoiceReversed(); // for reversed game
+    console.log("randomChoice");
+    console.log(randomChoice);
+    console.log("myoutputvalue");
+    console.log(myOutputValue);
+
+    // draw scenarios
+    if (guess == randomChoice) {
+      myOutputValue =
+        "The computer chose " +
+        randomChoice +
+        ". <br>" +
+        "You chose " +
+        guess +
+        ". <br><br>" +
+        "It's a draw! <br><br>" +
+        "Now you can type scissors, paper or rock to play another round! <br> You can also type Reversed Scissors, Reversed Paper, Reversed Rock to try the Reversed Game! <br><br> Your win-loss record is: <br> Your win record: " +
+        userWonCount +
+        `<br> CPU win record :` +
+        cpuWonCount;
+    }
+
+    // if input wins generateRandomChoice, output = you win
+    if (
+      // input = scissors, win randomchoice = paper
+      (guess == "scissors" && randomChoice == CHOICE_PAPER) ||
+      // guess paper, win randomchoice = rock
+      (guess == "paper" && randomChoice == CHOICE_ROCK) ||
+      // guess rock , win randomchoice = scissors
+      (guess == "rock" && randomChoice == CHOICE_SCISSORS)
+    ) {
+      userWonCount = userWonCount + 1; //user win + 1
+      myOutputValue =
+        "The computer chose " +
+        randomChoice +
+        ". <br>" +
+        "You chose " +
+        guess +
+        ". <br><br>" +
+        "Congratulations! It's a win! <br><br>" +
+        "Now you can type scissors, paper or rock to play another round! <br> You can also type Reversed Scissors, Reversed Paper, Reversed Rock to try the Reversed Game! <br><br> Your win-loss record is: <br> Your win record: " +
+        userWonCount +
+        `<br> CPU win record :` +
+        cpuWonCount;
+    }
+
+    // lose scenarios
+    if (
+      (guess == "scissors" && randomChoice == CHOICE_ROCK) ||
+      (guess == "paper" && randomChoice == CHOICE_SCISSORS) ||
+      (guess == "rock" && randomChoice == CHOICE_PAPER)
+    ) {
+      cpuWonCount = cpuWonCount + 1; // cpu win +1
+      myOutputValue =
+        "The computer chose " +
+        randomChoice +
+        ". <br>" +
+        "You chose " +
+        guess +
+        ". <br><br>" +
+        "You lose! Bummer. <br><br>" +
+        "Now you can type scissors, paper or rock to play another round! <br> You can also type Reversed Scissors, Reversed Paper, Reversed Rock to try the Reversed Game! <br><br> Your win-loss record is: <br> Your win record: " +
+        userWonCount +
+        `<br> CPU win record :` +
+        cpuWonCount;
+    }
+    console.log("myOutputValue for else");
+    console.log(myOutputValue);
```

Comment:
can remove console.log before sharing code to shorten code and increase readability